src/tests/SharpRaven.UnitTests/Data/HttpRequestBodyConverterTests.cs

```diff
@@ -34,15 +34,40 @@
 using System.IO;
 using System.Text;
 
+using Newtonsoft.Json;
 using NUnit.Framework;
 
 using SharpRaven.Data;
+using System.Collections;
 
 namespace SharpRaven.UnitTests.Data
 {
     [TestFixture]
     public class HttpRequestBodyConverterTests
     {
+        #region TestCaseData
```

Comment:
No, it is a my habit, usually i wrap private code on a dedicated region. Can i remove it if you want. If you are using a code convention let me know i will happy to follow it :)